instaloader/structures.py

```diff
@@ -1061,6 +1061,20 @@ def get_followers(self) -> NodeIterator['Profile']:
             'https://www.instagram.com/{0}/'.format(self.username),
         )
 
+    def get_followed_hashtags(self) -> Iterator['Hashtag']:
+        """
+        Retrive list of hashtags followed by given profile.
+        To use this, one needs to be logged in and private profiles has to be followed.
+        """
+        if not self._context.is_logged_in:
+            raise LoginRequiredException("--login required to get a profile's followers.")
+        self._obtain_metadata()
+        yield from (Hashtag(self._context, edge["node"]) for edge in
+            self._context.graphql_query("e6306cc3dbe69d6a82ef8b5f8654c50b",
+                                        {"id": str(self.userid)},
+                                        "https://www.instagram.com/{0}/"
+                                        .format(self.username))["data"]["user"]["edge_following_hashtag"]["edges"])
```

Comment:
This should rather return a NodeIterator, which does all the pagination handling and also supports resuming. If it's unclear how to create one, see the other invocations of NodeIterator or feel free to ask.

Reply:
That would be the case if the returned object supports pagination but its doesn't so it raises *KeyError* It's just 1 big list of nodes that contains a hashtag object.
![image](https://github.com/instaloader/instaloader/assets/44820142/d143f91b-5ae5-46b9-a705-a29887b57b3a)

**Still** i haven't came across a testing account with more than >55 followed hashtags but i'm certain it doesn't support pagination unlike returning an object of followers or followings (with 25 items)...

Reply:
Indeed, thanks for noting. In `upcoming/v4.10` I just pushed a fix to let the NodeIterator handle single paged responses like this one and it does not raise that KeyError anymore.

Reply:
Thanks for adding a fix, i have switched to branch upcoming/v4.10 and everything should be resolved now.